frontend/packages/console-dynamic-plugin-sdk/upgrade-PatternFly.md

```diff
@@ -0,0 +1,59 @@
+# PatternFly Upgrade Notes
+
+A dynamic plugin targets one or more versions of OpenShift Console. Each Console version supports
```

Comment:
```suggestion
A dynamic plugin targets one or more versions of OpenShift Console. Each version supports
```
We could just say version here. Console isn't really necessary based on previous sentence.